examples/kubernetes/kubeconfig

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: Config
+users:
+- name: konnectivity-server
+  user:
+    token: ${SERVER_TOKEN}
+clusters:
+- name: local
+  cluster:
+    insecure-skip-tls-verify: true
```

Comment:
This is only for the client side no?

Reply:
You are right. This means that the client won't verify the server's identity, so I think it's ok.

Reply:
This config will be used by konnectivity-server static pod for hitting local kube-apiserver. Without specifying  insecure-skip-tls-verify. tls will fail, since we need to pass ca.crt, but even by passing ca.crt tls fail, since ca.crt (at least in GKE) has cluster's IPs only (local host is not valid). Moreover, this is a pattern used by all static pods running in master.